test/harvester/events-reader.js

```diff
@@ -50,7 +50,8 @@ describe('onChange callback, event capture and at-least-once delivery semantics'
                 adapter: 'mongodb',
                 connectionString: process.argv[2] || process.env.MONGODB_URL || "‌mongodb://127.0.0.1:27017/test",
                 db: 'test',
-                inflect: true
+                inflect: true,
+                oplogConnectionString : (process.env.OPLOG_MONGODB_URL || process.argv[3]) + '?slaveOk=true'
```

Comment:
mongodb://127.0.0.1/local should be the final default, i think... it lines up with the default mongodb connection string above. And you may want to have the fall throughs line up in the same direction (process arg, then env, then hardcoded default)